api/api.go

```diff
@@ -124,13 +139,13 @@ func GetStateActivity(w http.ResponseWriter, r *http.Request, d *db.Database) {
 // StateCompare compares two versions ('from' and 'to') of a State
 func StateCompare(w http.ResponseWriter, r *http.Request, d *db.Database) {
 	w.Header().Set("Access-Control-Allow-Origin", "*")
-	st := util.TrimBasePath(r, "api/state/compare/")
+	lineage := util.TrimBasePath(r, "api/state/compare/")
```

Comment:
Should this be in `api/lineage/compare` now, or maybe in `api/lineage/<lineage>/compare`?

Reply:
> So I don't really know, for me this endpoint as well as api/state/compare/ are always used to perform operations on states, lineages are only one parameter used.
But if you find it more suitable to replace state by lineage, no problem.